codegen/src/main/resources/ServiceComb/consumer/microservice.mustache

```diff
@@ -5,7 +5,7 @@ service_description:
 #The microservice name should be unique within an application.
 #The name can contain digits, uppercase and lowercase letters, hyphens(-), underscores(_), and periods(.); and can neither start nor end with punctuations.
 #The naming rule is as follows: ^[a-zA-Z0-9]+$|^[a-zA-Z0-9][a-zA-Z0-9_-.]*[a-zA-Z0-9]$.
-  name: {{microserviceName}}-consumer
+  name: {{serviceId}}
```

Comment:
both of provider and consumer use the "serviceId" as value, it will generate the same service name while we 
 use ServiceType in all

Reply:
This problem does exist, thank you for your inspection. I will fix it in the next commit later.